$NetBSD: patch-scripts_miscellaneous_unpack.m,v 1.2 2022/10/24 18:02:19 adam Exp $

Unpack with bsdtar. if we get PaxHeaders.1234 files, we trip an octave-forge
sanity check

--- scripts/miscellaneous/unpack.m.orig	2022-07-28 13:08:26.000000000 +0000
+++ scripts/miscellaneous/unpack.m
@@ -200,15 +200,15 @@ function filelist = unpack (file, dir = 
                        'bzip2 -d -k -f "%s"', ...
                        @__parse_bzip2__, true};
     commandlist.bz = commandlist.bz2;
-    commandlist.tar = {'tar xvf "%s"', ...
-                       'tar xf "%s"', ...
+    commandlist.tar = {'bsdtar xvf "%s"', ...
+                       'bsdtar xf "%s"', ...
                        @__parse_tar__, false};
-    commandlist.targz = {'gzip -d -c "%s" | tar xvf -', ...
-                         'gzip -d -c "%s" | tar xf -', ...
+    commandlist.targz = {'gzip -d -c "%s" | bsdtar xvf -', ...
+                         'gzip -d -c "%s" | bsdtar xf -', ...
                          @__parse_tar__, false};
     commandlist.tgz = commandlist.targz;
-    commandlist.tarbz2 = {'bzip2 -d -c "%s" | tar xvf -', ...
-                          'bzip2 -d -c "%s" | tar xf -', ...
+    commandlist.tarbz2 = {'bzip2 -d -c "%s" | bsdtar xvf -', ...
+                          'bzip2 -d -c "%s" | bsdtar xf -', ...
                           @__parse_tar__, false};
     commandlist.tarbz = commandlist.tarbz2;
     commandlist.tbz2 = commandlist.tarbz2;
